examples/deno/index.ts

```diff
@@ -1,8 +1,10 @@
-import { serve } from 'https://deno.land/std@0.157.0/http/server.ts';
 import { yoga } from './yoga.ts';
 
-serve(yoga, {
-  onListen({ hostname, port }) {
-    console.log(`Listening on http://${hostname}:${port}${yoga.graphqlEndpoint}`);
+Deno.serve(
+  {
+    onListen({ hostname, port }) {
```

Comment:
Alternatively, this can be dropped. However, Deno prints:
```
Listening on http://0.0.0.0:8000/ (http://localhost:8000/)
```

whilst this custom log is more accurate with `/graphql` path mentioned:
```
Listening on http://0.0.0.0:8000/graphql
```